hs_core/management/commands/ingest_google_drive_shareable_link.py

```diff
@@ -0,0 +1,25 @@
+import gdown
+import tempfile
+import os
+from django.core.management.base import BaseCommand
+from django_irods.storage import IrodsStorage
+
+
+class Command(BaseCommand):
+    help = "Ingest a zipped bag archive into a resource"
+
+    def add_arguments(self, parser):
+
+        # a list of resource id's: none does nothing.
+        parser.add_argument('shareable_link', type=str)
+        parser.add_argument('resource_file_path', type=str)
+
+    def handle(self, *args, **options):
+        shareable_link = options['shareable_link']
+        resource_file_path = options['resource_file_path']
+        with tempfile.TemporaryDirectory() as temp_dir:
+            file_id = shareable_link.split('/')[-1].split('?')[0]
+            dfile = os.path.join(temp_dir, file_id)
+            gdown.download(url=shareable_link, output=dfile, fuzzy=True)
+            istorage = IrodsStorage()
+            istorage.saveFile(dfile, resource_file_path)
```

Comment:
this will iput the file into irods but we still need to ingest it into HS

Reply:
Right, but then the file listing patch in the other PR will detect it and create a reference. Maybe that is a little too lazy of me.

Reply:
Good point that it would work and in the case that we are uploading from Gdrive we probably don't have a lot of files so it's probably okay to be lazy.

Reply:
I guess I would at least add a comment to the management command explaining that we are relying on that lazy file ingestion

Reply:
I'll just add it in. It is only 2 lines, an import and calling the function.  Ultimately this management command won't be used. Once I show it works with a 25 GB file I'll wrap this in a API call so users can do it themselves.

Reply:
google drive says 4.5 hours to upload a 25 GB file so we won't have a test until the end of the day.